src/main/kotlin/io/github/crackthecodeabhi/kreds/commands/StringCommands.kt

```diff
@@ -59,10 +59,10 @@ internal interface BaseStringCommands {
     fun _set(key: String, value: String, setOption: SetOption?) =
         CommandExecution(SET, SimpleAndBulkStringCommandProcessor, *createArguments(
             key, value,
-            setOption?.exSeconds?.let { KeyValueArgument("EX", it.toString(10)) },
```

Comment:
Does this fix this below issue?

https://github.com/crackthecodeabhi/kreds/issues/11

Reply:
Yes it does